src/runtime/propensity-server.js

```diff
@@ -103,11 +103,18 @@ export class PropensityServer {
       method: 'GET',
       credentials: 'include',
     });
-    let userState = this.publicationId_ + ':' + state;
+    let stateParam = this.publicationId_ + ':' + state;
+    let extrainfoParam = '&extrainfo=';
     if (productsOrSkus) {
-      userState = userState + ':' + encodeURIComponent(productsOrSkus);
+      const skus = encodeURIComponent(productsOrSkus);
```

Comment:
Why does the variable name switch from `productsOrSkus` to just `skus` here?

Reply:
Its just a local const so it wasn't as important to have a clear name - I removed the const anyway because we don't need to send the value twice anyway (as they aren't using the JSON object right now).

Reply:
Could it still contain products then? Looks like the `const` is still around

Reply:
I just hadn't updated the code yet